plugins/auth/login.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/* eslint no-param-reassign: ["error", { "props": false }]*/
+/* eslint no-param-reassign: ["error", { "props": false }] */
```

Comment:
@FenrirUnbound just tried it after removing that comment and nope I don't get a linting error. I'll make the change